File: ClientApp/app/models/Timesheet.ts
import { Carer } from './Carer';
import { CarerContract } from './Contract';
import { Availability } from './Availability';
import { CarerBooking } from './Booking';
import { Shift } from './Shift';
import { Adjustment } from './Adjustment';

export class Timesheet {
	public carerCode: number;
	public weekCommencing: Date;
	public carer: Carer;
	public contracts: CarerContract[];
	public scheduledAvailability: Availability[];
	public actualAvailability: Availability[];
	public bookings: CarerBooking[];
	public shifts: Shift[];
	public adjustments: Adjustment[];
}

File: Data/Models/Timesheet.cs
namespace Blackwood.Access.Models {
    using System;
    using System.Collections.Generic;

    public class Timesheet {
		
		public int CarerCode { get; set; }
		public DateTime WeekCommencing { get;set; }
		public Carer Carer { get; set; }
		public ICollection<CarerContract> Contracts { get; set; }
		public ICollection<Availability> ScheduledAvailability { get; set; }
		public ICollection<Availability> ActualAvailability { get; set; }
		public ICollection<CarerBooking> Bookings { get; set; }

	}
}